examples/lush_tutorial.lua

```diff
@@ -24,11 +23,9 @@
 -- Look out! We're currently working in a temporary buffer. If you want to
 -- keep anything you do here, make sure you use `:saveas <your filename>`!
 -- Just using `:w` wont work! It's probably best to quickly run through this
--- file, then discard it and see CREATE.md for more unstructions about setting
+-- file, then discard it and see CREATE.md for more instructions about setting
 -- up your own theme.
 
-
-
 -- ###
```

Comment:
The whitespace breaks were intended to separate the "learning sections" a bit clearer. Did you find them a bit too distracting? I know we also have the "heading blocks" too but without the whitespace it felt kind of dense when I was writing it out.

Reply:
My apologies, I thought clearing out the blank lines would be condensing it out a bit, but I'm gonna revert it back, though how about the first line?

Reply:
I thought the ascii graphics looked a bit cramped without it :)

Reply:
Fair point